apps/stats-dapp/src/constants/index.ts

```diff
@@ -3,3 +3,5 @@ const isDev = process.env.NODE_ENV === 'development';
 export const defaultEndpoint = isDev
   ? 'https://subquery-dev.webb.tools/graphql'
   : 'https://subquery-dev.webb.tools/graphql';
+
+export const polkadotProviderEndpoint = 'wss://arana-alpha-1.webb.tools';
```

Comment:
@dutterbutter @devpavan04  this fine, but we should query the specName or the chain from the subquery node and make sure we are using the same endpoint as the subquery node so we don't end with different sources

```graphql
query{
  _metadata{
    chain
    specName
    
  }
}
```

```json
{
  "data": {
    "_metadata": {
      "chain": "Arana Alpha",
      "specName": "tangle-standalone"
    }
  }
}
```

Reply:
Yes let's avoid relying on hardcoded values as the stats dApp frontend should be usable for multiple webb networks.